commands/logs_test.go

```diff
@@ -0,0 +1,26 @@
+package commands
+
+import (
+	"bytes"
+	"io/ioutil"
+	"testing"
+
+	"github.com/docker/docker/pkg/stdcopy"
+	"github.com/mesg-foundation/core/container"
+)
+
+func TestLogsCmdRunE(t *testing.T) {
+	m := newMockExecutor()
+	c := newLogsCmd(m)
+
+	// create reader for docker stdcopy
+	buf := new(bytes.Buffer)
+	w := stdcopy.NewStdWriter(buf, stdcopy.Stdout)
+	w.Write([]byte("core: 2018-01-01 log\n"))
+
+	m.On("Status").Return(container.RUNNING, nil)
+	m.On("Logs").Return(ioutil.NopCloser(buf), nil)
+	c.cmd.Execute()
+
+	m.AssertExpectations(t)
```

Comment:
this line can be removed, we don't have any expected arguments while calling the methods. or maybe always keep this to not forget to put it in needed places? 

/cc @mesg-foundation/core

Reply:
It isn't only for arguments, it is also for the number of calls (for example if Logs wasn't more times etc...)

Reply:
yes, i see now.